### PR TITLE
Add calendar url to profile page

### PR DESCRIPTION
**Pull request by asimonson1125**
I did my best but uhh the whole save thing doesn't really work.  
Used account.get('icallink') to get pre-added calendars but to save I'd need an account.icallink attribute which doesn't exist (+ edit permissions).

### CODE REVIEW

**Comment by mxmeinhold** [comment]
The attribute doesn't need to exist for you to write to it. The cshLDAP library will map `account.icallink` to `icallink` in ldap (you should also be able to do `account.icallink` instead of `account.get(...)`)

This is done through `__getattr__` and `__setattr__`, which define how a python class handles reads and writes to attributes
Source code is here, if you're curious https://github.com/liam-middlebrook/csh_ldap/blob/29f26006fcb0717473ae6e791faf5a79560c8585/csh_ldap/member.py#L93-L159
and the python docs on the topic are here https://docs.python.org/3/reference/datamodel.html#customizing-attribute-access

**Comment by asimonson1125** [comment]
woah rad, always wondered what python did with those.
Just updated a line to reflect that, still don't know if the saving part actually works, but it's basically a carbon copy of preexisting attributes.

**Comment by asimonson1125** [comment]
Got this deployed on profiles-dev, looks like it's working.

File: profiles/ldap.py [hunk 1]
```diff
@@ -329,6 +332,11 @@ def ldap_get_roomnumber(account):
     except AttributeError:
         return ""
 
+def ldap_get_calendar(account):
+    try:
+        return account.icallink
+    except KeyError:
```
Comment (WillNilges) [inline profiles/ldap.py:338]: This smells like it fails silently. Maybe log this?

File: profiles/utils.py [hunk 2]
```diff
@@ -12,18 +12,20 @@
...
+                           ldap_get_groups,
+                           ldap_is_rtp)
 
 
 def before_request(func):
     @wraps(func)
     def wrapped_function(*args, **kwargs):
-        git_revision = subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD']).decode('utf-8').rstrip()
+        git_revision = subprocess.check_output(
+            ['git', 'rev-parse', '--short', 'HEAD']).decode('utf-8').rstrip()
```
Comment (WillNilges) [inline profiles/utils.py:28]: Lol this format is funkyyyyy

**Review by WillNilges — APPROVED** [review]
:shipit:

**Pull request merged**